stories/core/ComboBox.stories.tsx

```diff
@@ -392,16 +398,22 @@ export const WithStatus: Story<Partial<ComboBoxProps<string>>> = (args) => {
   const options = React.useMemo(() => countriesList, []);
 
   return (
-    <ComboBox
-      options={options}
-      inputProps={{ placeholder: 'Select a country' }}
-      onChange={(value: string) => action(value ?? '')()}
-      status='negative'
-      {...args}
-    />
+    <InputGroup
+      status={args.status ?? 'negative'}
+      message='Country is required.'
+    >
+      <Label htmlFor='country-input'>Country</Label>
+      <ComboBox
+        options={options}
+        inputProps={{ id: 'country-input', placeholder: 'Select a country' }}
+        onChange={(value: string) => action(value ?? '')()}
+        status='negative'
```

Comment:
I think this hardcoded `status` is not needed anymore.

Reply:
What do you mean by "not needed"? Is it because storybook args will override it? Then I left it on purpose for better readability.
Or is it because input group now has it? If thats, then it is needed, because without it Combobox will not be styled with the red bottom border.

Reply:
Better readability is good purpose.
Combobox is styled with red bottom border even without this `status: 'negative'`.